### PR TITLE
Updating error message for node check while building cool for iOS.

### PR DESCRIPTION
**Pull request by k-dasari**
* Target version: master 

### Summary
This change warns the user to install node and to update proper path in make file while building cool for iOS in Mac machines.

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [ ] All commits have Change-Id
- [x] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by vmiklos — COMMENTED** [review]
Looks OK in general, I think only Tor had this setup of building without node on macOS (doing the JS piece on Linux, to keep his macOS "clean"), and he's not contributing anymore.

CC @Minion3665 in case he would have some objections.

One easy, but important part: could you do `git commit --amend -s` to sign off your commit and force-push to this PR to add the missing sign-off? See README.CONTRIBUTING.md on what does this mean. Thanks!

**Review by Minion3665 — COMMENTED** [review]
I have no objections once the sign-off and change-id are done

**Comment by pedropintosilva** [comment]
Welcome and thank you for taking the time to help out!

I have added reviewers but meanwhile please correct the 2 falling tests

To fix DCO please follow the instrucations https://github.com/CollaboraOnline/online/runs/25267664946

To add the missing change-id

File: configure.ac
Comment (timar) [inline configure.ac:1397]: I don't think Makefile with `no` would be generated. I think configure script will stop at error. Hence the `once installed replace 'no' with actual node path in Makefile` part is not necessary.

Comment (k-dasari) [reply configure.ac:1397]: I got below error even after installing the node
`Checking for CSS errors...
/bin/sh: no: command not found
make[1]: *** [Makefile:1100: dist/bundle.css] Error 127
make[1]: Leaving directory '/Users/karthik/Desktop/online/browser'
make: *** [Makefile:709: all] Error 2`

And error got resolved after replacing `no` with actual path of the node in make file. Hence made these changes.

Comment (vmiklos) [reply configure.ac:1397]: Right, you need to `make distclean` after installing node, configure assumes the system doesn't change after running once. In any case, could you please amend your commit message to include the signed-off-by line as requested above? We can take care of the rest. Thanks.

Comment (k-dasari) [reply configure.ac:1397]: Done

**Review by vmiklos — APPROVED** [review]
Looks good to me now, let's just wait for CI.

**Comment by vmiklos** [comment]
<https://cpci.cbg.collabora.co.uk:8080/job/github_online_master_debug_vs_co-24.04_cypress_multi_user/993/console> says sheet_operations_spec.js failed, retry.

**Pull request merged**
